bentoml/db.py

```diff
@@ -33,9 +34,19 @@
 
 def init_db(db_url):
     # Use default config if not provided
-    engine = create_engine(
-        db_url, echo=False, connect_args={'check_same_thread': False}
-    )
+    # we have to parse the db url. Depends on what type of it,
+    if is_sqlite_db(db_url):
+        engine = create_engine(
+            db_url, echo=False, connect_args={'check_same_thread': False}
+        )
+    elif is_postgres_db(db_url):
+        if not is_postgres_db_name_exists(db_url):
+            db_url = os.path.join(db_url, 'bentoml')
```

Comment:
just raise and show an error message when the db does not exist.  also should not use os.path.join here.